commands/register.go

```diff
@@ -149,18 +149,15 @@ func messageReaction(s *discordgo.Session, m *discordgo.MessageReactionAdd) {
 		return
 	}
 	react := Reaction(m.MessageReaction.Emoji.Name)
-	data, ok := reactionMap[m.MessageID]
-	if ok {
-		switch content := data.(type) {
-		case *api.Event:
+	if data, ok := reactionMap[m.MessageID]; ok {
+		if content, ok := data.(api.Entry); ok {
 			switch react {
 			case twitter:
-				fmt.Println(content.Description)
 				mediaIds := []int64{}
-				if content.Image != nil {
+				if content.GetImage() != nil {
 					// Contains image
 					// Upload image
-					getImage, err := http.Get(content.Image.Request.URL.String())
+					getImage, err := http.Get(content.GetImage().Request.URL.String())
 					if err != nil {
 						log.Error(err.Error())
```

Comment:
`log.WithError(err).Error("failed to perform epic reddit gamer keanu reeves big chungus moment")`